compiler/luci-interpreter/include/luci_interpreter/core/Tensor.h

```diff
@@ -60,6 +60,17 @@ class Shape
     return result;
   }
 
+  // TODO Replace num_elements
+  int64_t large_num_elements() const
```

Comment:
@jinevening Why do you use signed int here? Does `large_num_elements()` returns negative number? It could be happen when some of `dim` has `-1`, which means `undefined-yet`.

Reply:
Just because existing `num_elements()` returns signed int (`int32_t`). I think it is good to use `uint64_t`!